apps/docs/src/app/core/component-docs/action-sheet/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './examples/action-sheet-compact/action-sheet-compact-example.component';
```

Comment:
Lets just do it like for example here: [link](https://github.com/SAP/fundamental-ngx/tree/b9b4d46b322fe6ab247eff40b47b0f7d345c91d2/apps/docs/src/app/core/component-docs/busy-indicator/examples)